index.md

```diff
@@ -277,100 +277,84 @@ href="{{site.swc_site}}">Software Carpentry</a>, <a href="{{site.dc_site}}">Data
 {% elsif site.incubator_pre_survey or site.incubator_post_survey %}
 <div class="alert alert-danger">
-WARNING: you have defined custom pre- and/or post-survey links for
-a workshop not configured for The Carpentries Incubator
-(the value of `curriculum` is not set to `incubator` in `_config.yml`).
-Please comment out the `incubator_pre_survey` and `incubator_post_survey` fields
-in `_config.yml` or, if this workshop is teaching a lesson in the Incubator,
-change the value of `carpentry` to `incubator`.
+УВАГА: ви визначили власні посилання до та/або після опитування для семінару, який не налаштований для The Carpentries Incubator (значення 'curriculum' не встановлено на 'incubator' у '_config.yml'). Будь ласка, закоментуйте поля «incubator_pre_survey» та «incubator_post_survey» у «_config.yml» або, якщо цей семінар проводить урок в Incubator, змініть значення «carpentry» на incubator.
 </div>
 {% else %}
-<p><a href="{{ site.pre_survey }}{{ site.github.project_title }}">Pre-workshop Survey</a></p>
-<p><a href="{{ site.post_survey }}{{ site.github.project_title }}">Post-workshop Survey</a></p>
+<p><a href="{{ site.pre_survey }}{{ site.github.project_title }}">Опитування перед семінаром</a></p>
+<p><a href="{{ site.post_survey }}{{ site.github.project_title }}">Опитування після семінару</a></p>
 {% endif %}
 
 <hr/>
 
 
 {% comment %}
-SCHEDULE
-
-Show the workshop's schedule.
-
-Small changes to the schedule can be made by modifying the
-`schedule.html` found in the `_includes` folder for your
-workshop type (`swc`, `lc`, or `dc`). Edit the items and
-times in the table to match your plans. You may also want to
-change 'Day 1' and 'Day 2' to be actual dates or days of the
-week.
-
-For larger changes, a blank template for a 4-day workshop
-(useful for online teaching for instance) can be found in
-`_includes/custom-schedule.html`. Add the times, and what
-you will be teaching to this file. You may also want to add
-rows to the table if you wish to break down the schedule
-further. To use this custom schedule here, replace the block
-of code below the Schedule `<h2>` header below with
-`{% include custom-schedule.html %}`.
-{% endcomment %}
+РОЗКЛАД
+
+Покажіть розклад семінару.
+
+Невеликі зміни в розклад можна внести, змінивши «schedule.html», який знаходиться в папці «_includes» для вашого типу семінару («swc», «lc» або «dc»). Відредагуйте пункти та час у таблиці відповідно до своїх планів. Ви також можете змінити поля "День 1" і "День 2" на фактичні дати або дні тижня.
+
+  Для більших змін порожній шаблон для 4-денного семінару
+  (корисний, наприклад, для онлайн-викладання) можна знайти в розділі
+  `_includes/custom-schedule.html`. Додайте до цього файлу час і те, чого ви будете навчати. Ви також можете додати рядки до таблиці, якщо хочете розбити розклад далі. Щоб використовувати цей користувацький розклад тут, замініть блок коду під заголовком Розклад`<h2>` нижче на `{% include custom-schedule.html %}`.
```

Comment:
Not very critical, but using hard line breaks in the long run makes inspecting changes and resolving conflicts easier. Try to not to use long lines in subsequent translations please.